### ISSUE TITLE
{% include '@namespace/btn/btn.twig' with {...} %} dosen't work from kss-node v.3.0.0-beta16

### ISSUE THREAD

**Issue by marceldobrica** [kss-node/kss-node#509]
Current version, beta23, works with {% include '../btn/btn.twig ...%} but starting from version beta19 {% include @namespace/btn/btn.twig ...%} dosen't work anymore. Last version on which I was able to use the namespace feature was beta16.
I think it should be a test for {% include '@namespace/... %}... for me it is the most important feature. It is difficult to have include without namespaces and to change all the time twig components...

I don't know if it is kss-node issue .... I only use "kss": "3.0.0-beta.16", in my package.json to be able to use {% include @namespace/...%}... any above version dosen't work...

I found this bug testing  https://github.com/mediacurrent/theme_generator_8 and I have also mentioned  the issue there ....

**Comment by RobLoach** [comment]
I've found I needed to bring in the latest from Twig to fix this...

```
  "dependencies": {
    "twig": "RobLoach/twig.js#compile"
  }
```


Resolutions may work too...
```
  "resolutions": {
    "twig": "RobLoach/twig.js#compile"
  }
```

Feel free to encourage a Twig release over at https://github.com/twigjs/twig.js/issues/582 .

**Comment by mariohernandez** [comment]
Duplicate issue here https://github.com/kss-node/kss-node/issues/467.  Adding these dependencies seems to fix the issue which seems to be related to twig.js used in KSS Node.
`    "kss": "^3.0.0-beta.23",
    "twig": "RobLoach/twig.js#compile"`

**Comment by RobLoach** [comment]
Twig 1.13.0 is out! https://github.com/twigjs/twig.js/releases/tag/v1.13.0

**Comment by marceldobrica** [comment]
In media current theme generator, now with Twig 1.13.0 svg icons are imported converting tags as htmlentities...: < is converted in &lt; and so on...

 &lt;svg aria-hidden="true" role="img" class="kss-icons__icon" xmlns="http://www.w3.org/2000/svg" width="11" height="20" viewBox="0 0 11 20"&gt;&lt;path d="M10.703.134V3.08H8.951q-.96 0-1.295.402t-.335 1.205v2.109h3.27l-.435 3.304H7.321v8.471H3.906V10.1H1.06V6.796h2.846V4.363q0-2.076 1.161-3.22T8.159-.001q1.641 0 2.545.134z"/&gt;&lt;/svg&gt;....

so is still better to use 
"devDependencies": {
    "kss": "3.0.0-beta.23",
    "twig": "RobLoach/twig.js#compile"
  }

**Comment by RobLoach** [comment]
Is `autoescape` being set to `true` somewhere? Found that in KSS. Make sure it's set to false to allow unescaped SVGs.

**Comment by marceldobrica** [comment]
Thank you for your support RobLoach. Unfortunately I'm not able to find a solution and I need more help. I'm trying to insert a twig in another one...
    {% if btn_text %}
        {% include '@namespace/btn/btn.twig' with {'text': btn_text} only %}
    {% endif %}

If I'm using  
"devDependencies": {
"kss": "3.0.0-beta.23", 
"twig": "RobLoach/twig.js#compile"
} works perfect!... 
and options.autoescape is set to true in kss_builder_base_twig.js ....

If I'm using "devDependencies": {
"kss": "3.0.0-beta.23"
} ... doesn't work.... 

If I'm using "devDependencies": {
"kss": "3.0.0-beta.24",
"twig": "RobLoach/twig.js#compile"
} ... doesn't work.

If I'm using "devDependencies": {
"kss": "3.0.0-beta.24"
} and set options.autoescape = false in kss_builder_base_twig.js (line 109) it works.... but probably I should not make such a change in a node module .... probably I should make a change in builder class but what should I write....  

/**
 * A kss-node builder that takes input files and builds a style guide using Twig
 * templates.
 */
class KssBuilderTwig extends KssBuilderBaseTwig {
  /**
   * Create a builder object.
   */
  constructor() {
    // First call the constructor of KssBuilderBaseTwig.
    super();

    // Then tell kss which Yargs-like options this builder adds.
    this.addOptionDefinitions({
      title: {
        group: 'Style guide:',
        string: true,
        multiple: false,
        describe: 'Title of the style guide',
        default: 'KSS Style Guide'
      }
    });
  }
}

**Comment by RobLoach** [comment]
I think I've made that change as part of KSS `3.0.0-beta.25`. Autoescape is problematic, and we should likely be using it, but for now keep it disabled.

**Comment by marceldobrica** [comment]
Thank you RobLoach !
I think this issue was solved with twig 1.13.0 and KSS 3.0.0-beta.25 and is no more useful for anyone so I'll close it.
Was a pleasure to meet you online! 
Best regards and all the best !

**Comment by RobLoach** [comment]
Thanks a lot for all the debugging :+1:
